packages/trpc-interface/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@webstudio-is/trpc-interface",
+  "version": "0.20.0",
+  "description": "Webstudio Trpc Interface",
```

```suggestion
  "description": "Webstudio TRPC Interface",
```

When using english - TRPC is upper case because its abbreviation.
When using code - Trpc is camel case to follow a consistent code-convention